src/Pages.js

```diff
@@ -0,0 +1,81 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {loadJson, sendJson} from './sendJson';
+import {useParams} from 'react-router-dom';
+
+const loadOnePage = async (id, onLoad, setError) => {
```

Comment:
It's bad to have duplicated code. Can you make a single function in a different file that replaces this one and any like it? I see one in AdminPage.js, Book.js, CreateBookPage.js, and here.